.github/workflows/lint.yml

```diff
@@ -0,0 +1,59 @@
+name: Lint & Tidy
+on:
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  RUST_BACKTRACE: 1
+  SHELL: /bin/bash
+  SCCACHE_GHA_ENABLED: "true"
+  RUSTC_WRAPPER: "sccache"
+  CCACHE: "sccache"
```

Comment:
I think these aren't used currently...?

Reply:
I think they are for clippy: https://github.com/servo/servo/actions/runs/10508645663/job/29112990211?pr=33150#step:23:5